Project/demo/src/main/java/com/example/demo/model/UserRepository.java

```diff
@@ -0,0 +1,14 @@
+    package com.example.demo.model;
```

Comment:
UserRepository should be in separate repository directory in order to follow MVC model